File: src/main/java/com/extendedae_plus/integration/recipeViewer/emi/EaepEmiPlugin.java
package com.extendedae_plus.integration.recipeViewer.emi;

import com.extendedae_plus.integration.recipeViewer.emi.emiIntegrations.EmiStackConverters;
import com.extendedae_plus.integration.recipeViewer.emi.emiIntegrations.converter.EmiFluidStackConverter;
import com.extendedae_plus.integration.recipeViewer.emi.emiIntegrations.converter.EmiItemStackConverter;
import dev.emi.emi.api.EmiEntrypoint;
import dev.emi.emi.api.EmiPlugin;
import dev.emi.emi.api.EmiRegistry;

@EmiEntrypoint
public class EaepEmiPlugin implements EmiPlugin {
    @Override
    public void register(EmiRegistry registry) {
        EmiStackConverters.register(new EmiItemStackConverter());
        EmiStackConverters.register(new EmiFluidStackConverter());
    }
}

File: src/main/kotlin/com/extendedae_plus/integration/impl/recipeViewer/emi/EAEPEmiPlugin.kt
package com.extendedae_plus.integration.impl.recipeViewer.emi

import com.extendedae_plus.mixin.impl.ExclusionZoneScalingButton
import dev.emi.emi.api.EmiEntrypoint
import dev.emi.emi.api.EmiPlugin
import dev.emi.emi.api.EmiRegistry

@EmiEntrypoint
class EAEPEmiPlugin : EmiPlugin {
    override fun register(registry: EmiRegistry) {
        registry.addGenericExclusionArea(ExclusionZoneScalingButton)
    }
}
